File: embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/LogType.java
package io.embrace.android.embracesdk;

/**
 * Deprecated: use Severity instead. This enum is deprecated and will be removed in
 * a future release.
 *
 * Will flag the message as one of info, warning, or error for filtering on the dashboard
 */
@Deprecated
public enum LogType {
    INFO,
    WARNING,
    ERROR;

    final EventType toEventType() {
        switch (this) {
            case WARNING:
                return EventType.WARNING_LOG;
            case ERROR:
                return EventType.ERROR_LOG;
            default:
                return EventType.INFO_LOG;
        }
    }
}

File: embrace-internal-api/src/main/kotlin/io/embrace/android/embracesdk/LogType.kt
package io.embrace.android.embracesdk

import androidx.annotation.Keep

/**
 * Deprecated: use Severity instead. This enum is deprecated and will be removed in
 * a future release.
 *
 * Will flag the message as one of info, warning, or error for filtering on the dashboard
 */
@Keep
@Deprecated("")
enum class LogType {
    INFO,
    WARNING,
    ERROR
}
